lib/apiservers/engine/backends/container.go

```diff
@@ -988,6 +996,33 @@ func (c *Container) ContainerAttach(name string, ca *backend.ContainerAttachConf
 		}
 	}
 
+	client := c.containerProxy.Client()
+	handle, err := c.Handle(name, paramName)
+	if err != nil {
+		return err
+	}
+
+	_, err = client.Interaction.InteractionBind(interaction.NewInteractionBindParamsWithContext(ctx).
```

Comment:
We should be getting the returned handle here and using that in the call to Commit